pkgs/development/python-modules/nassl/default.nix

```
@@ -108,18 +112,22 @@ buildPythonPackage rec {
     invoke package.wheel
   '';
 
+  doCheck = true;
+
+  pythonImportsCheck = [ "nassl" ];
+
   checkInputs = [ pytestCheckHook ];
 
-  checkPhase = ''
-    # Skip online tests
-    pytest -k 'not Online'
-  '';
+  pytestFlagsArray = [
+    "-k"
+    "'not Online'"
+  ];
```

This is the more succinct way to express this, but essentially does the same thing.
```suggestion
  disabledTests = [
    "Online"
  ];
```

---

Thanks for the suggestion! Changed it accordingly 🙂